bitwarden_license/test/CmmCore.Test/Services/ProviderServiceTests.cs

```diff
@@ -197,7 +200,12 @@ public async Task InviteUserAsync_ProviderIdIsInvalid_Throws(Provider provider,
             var providerUserRepository = sutProvider.GetDependency<IProviderUserRepository>();
             providerUserRepository.GetManyAsync(default).ReturnsForAnyArgs(providerUsers.ToList());
 
-            var result = await sutProvider.Sut.ResendInvitesAsync(provider.Id, default, providerUsers.Select(pu => pu.Id));
+            var invite = new ProviderUserInvite<Guid>
+            {
+                UserIdentifiers = providerUsers.Select(pu => pu.Id),
+                ProviderId = provider.Id
+            };
+            var result = await sutProvider.Sut.ResendInvitesAsync(invite);
             Assert.True(result.All(r => r.Item2 == ""));
         }
 
```

Comment:
Maybe a few new tests for bad permissions on invites?

Reply:
Added